pkg/antctl/raw/set/flowaggregator/command_test.go

```diff
@@ -0,0 +1,294 @@
+    server: https://localhost
+  name: fake-cluster
+contexts:
+- context:
+    cluster:  fake-cluster
+    user:  user-id
+  name:  fake-cluster
+current-context:  fake-cluster
+kind: Config`),
+		},
+	}
+
+	for _, tc := range tcs {
+		t.Run(tc.name, func(t *testing.T) {
+			fakeKubeconfig, err := os.CreateTemp("", "fakeKubeconfig")
+			if err != nil {
+				log.Fatal(err)
+			}
+			defer os.Remove(fakeKubeconfig.Name())
+			fakeKubeconfig.Write(tc.fakeConfigs)
+			if cmd.Flag("kubeconfig") != nil {
+				cmd.Flag("kubeconfig").Value.Set(fakeKubeconfig.Name())
+			} else {
+				cmd.Flags().String("kubeconfig", fakeKubeconfig.Name(), "path of kubeconfig")
+			}
+			_, err = getk8sClient(cmd)
+			if tc.expectedErr == "" {
+				assert.NoError(t, err)
+			} else {
+				assert.Contains(t, err.Error(), tc.expectedErr)
+			}
+		})
+	}
+}
+
+func TestUpdateRunE(t *testing.T) {
+	cmd := NewFlowAggregatorSetCommand()
+	fakeConfigMap := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "testConfigMap",
+			Namespace: "flow-aggregator",
+		},
+	}
+
+	k8sClient := fake.NewSimpleClientset(fakeConfigMap)
+	tcs := []struct {
+		name           string
+		configMapName  string
+		podNamespace   string
+		args           []string
+		expectedErr    string
+		expectedOutput map[string]string
+	}{
+		{
+			name:           "valid ConfigMap and nil args",
+			configMapName:  "testConfigMap",
+			podNamespace:   "flow-aggregator",
+			expectedOutput: map[string]string{"flow-aggregator.conf": "{}\n"},
```

Comment:
why the content is sometimes json sometimes yaml?

Reply:
the output is in yaml format.